tests/functional/oauth_webchannel.js

```diff
@@ -96,6 +96,7 @@ define([
 
     'signin a verified account using an oauth app': function () {
       var self = this;
+      self.shouldCloseTab = true;
```

Comment:
All other "helpers" are self-contained. i.e they accept `options`, such as 
https://github.com/mozilla/fxa-content-server/blob/master/tests/functional/lib/helpers.js#L28

They do not depend on some global `Test` state. Should we use `options` instead of setting these variables.

cc @shane-tomlinson